load-generator/build.gradle

```diff
@@ -32,9 +32,10 @@ dependencies {
 
     implementation platform("io.opentelemetry:opentelemetry-bom:1.12.0")
     implementation "io.opentelemetry:opentelemetry-api"
-    implementation "io.opentelemetry:opentelemetry-sdk-metrics:1.4.1-alpha"
-    implementation "io.opentelemetry:opentelemetry-exporter-otlp:1.10.0"
-    implementation "io.opentelemetry:opentelemetry-exporter-otlp-metrics:1.4.1-alpha"
+    implementation "io.opentelemetry:opentelemetry-sdk-metrics:1.18.0"
```

Comment:
Since you are use a bom, you don't need to define version for the dependencies:

https://blog.mrhaki.com/2019/04/gradle-goodness-use-bill-of-materials.html

You can just update the version of the bom: 
```
   implementation platform("io.opentelemetry:opentelemetry-bom:1.18.0")
   implementation "io.opentelemetry:opentelemetry-api"
   implementation "io.opentelemetry:opentelemetry-sdk-metrics"
....
```

Reply:
Thanks. updated accordingly.